DPG/Tasks/AOTTrack/qaMatchEff.cxx

```diff
@@ -87,14 +87,15 @@ struct qaMatchEff {
   Configurable<bool> isPbPb{"isPbPb", false, "Boolean to tag if the data is PbPb collisions. If false, it is pp"};
   Configurable<bool> isEnableEventSelection{"isEnableEventSelection", true, "Boolean to switch the event selection on/off."};
   Configurable<bool> isCentralityRequired{"isCentralityRequired", false, "Boolean to switch the centrality selection on/off."};
+  Configurable<bool> isRejectNearByEvent{"isRejectNearByEvent", false, "Boolean to switch the rejection of near by events on/off."};
   Configurable<bool> isEnableOccupancyCut{"isEnableOccupancyCut", false, "Boolean to switch the occupancy cut on/off."};
   struct : ConfigurableGroup {
     Configurable<float> centralityMinCut{"centralityMinCut", 0.0f, "Minimum centrality"};
     Configurable<float> centralityMaxCut{"centralityMaxCut", 100.0f, "Maximum centrality"};
   } centralityCuts;
   struct : ConfigurableGroup {
-    Configurable<int> minTracksInTimeRange{"minTracksInTimeRange", 0, "Minimum number of tracks in the time range"};
-    Configurable<int> maxTracksInTimeRange{"maxTracksInTimeRange", 1000, "Maximum number of tracks in the time range"};
+    Configurable<float> minTracksInTimeRange{"minTracksInTimeRange", 0.0f, "Minimum number of tracks in the time range"};
+    Configurable<float> maxTracksInTimeRange{"maxTracksInTimeRange", 999999.9f, "Maximum number of tracks in the time range"};
```

Comment:
what's the reason to change the data type here?

Reply:
The default type of occupancy should indeed be int, and this has been corrected in the latest commit. 
The default maximum cut was too small and has been changed to a larger value.